http-cors/src/main/scala/org/apache/pekko/http/cors/scaladsl/model/HttpHeaderRange.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.pekko.http.cors.scaladsl.model
+
+import java.util.Locale
+
+import org.apache.pekko.http.cors.javadsl
+
+import scala.collection.immutable.Seq
+
+abstract class HttpHeaderRange extends javadsl.model.HttpHeaderRange
+
+object HttpHeaderRange {
+  case object `*` extends HttpHeaderRange {
+    def matches(header: String) = true
+  }
+
+  final case class Default(headers: Seq[String]) extends HttpHeaderRange {
+    val lowercaseHeaders: Seq[String] = headers.map(_.toLowerCase(Locale.ROOT))
+    def matches(header: String): Boolean = lowercaseHeaders contains header.toLowerCase(Locale.ROOT)
```

Comment:
suggestion: use `org.apache.pekko.util.Helpers.toRootLowerCase` instead. Not sure there is a big advantage, just consistency (I would have not created this helper in the first place)

Reply:
Fixed.